pkg/morph/client/notary.go

```diff
@@ -3,6 +3,7 @@ package client
 import (
 	"errors"
 	"fmt"
+	"math/rand"
```

Comment:
Use math rand with cryptographic source from util package there. I don't like any use of default `math/rand` where random values matter.

```go
import "github.com/nspcc-dev/neofs-node/pkg/util/rand"
rand.New().Uint32()

Reply:
ok, is it ok to call `New` every time I need rand number or it can be global var in `client` pgk?

Reply:
Sure it is better to be a global singleton.

Reply:
changed dependency